spec/Appendix A -- Persisted Documents.md

```diff
@@ -95,6 +95,35 @@ implementation specific.
 Note: A 32 character hexadecimal _custom document identifier_ is likely to be an
 MD5 hash of the GraphQL document, as traditionally used by Relay.
 
+### Document identifier syntax
+
+DocumentIdentifier ::
+
+- PrefixedDocumentIdentifier
+- CustomDocumentIdentifier
+
+PrefixedDocumentIdentifier ::
+
+- Sha256HexDocumentIdentifier
+- OtherPrefixedDocumentIdentifier
+
+Sha256HexDocumentIdentifier :: sha256 `:` Sha256Checksum
+
+Sha256Checksum :: 64 LowerCaseHexDigit
+
+LowerCaseHexDigit :: one of
+
+- `0` `1` `2` `3` `4` `5` `6` `7` `8` `9`
+- `a` `b` `c` `d` `e` `f`
+
+OtherPrefixedDocumentIdentifier :: x- IdentifierCharacter+ `:` IdentifierCharacter+
+
+CustomDocumentIdentifier :: SourceCharacter+
+
+IdentifierCharacter :: SourceCharacter but not `:`
+
+SourceCharacter :: "Any Unicode scalar value"
```

Comment:
```suggestion
SourceCharacter :: Any Unicode scalar value
```

Reply:
I took over this one from the main spec ([source](https://github.com/graphql/graphql-spec/blob/3adfcca73644234fbbbb062a5cec9e7703419a9f/spec/Appendix%20B%20--%20Grammar%20Summary.md?plain=1#L5))

Overall, agree more consistency would be good here. Backticks for terms that are actually part of the syntax as-is and no quote/tick for sentences that are explicative but should not appear in the constructed element?

Reply:
I 've pushed backticks and no-quotes in https://github.com/graphql/graphql-over-http/pull/296/commits/81fa52953cbafe0e3f9a8c41a39f707cd12b68fc